CONTRIBUTING.adoc

```diff
@@ -50,101 +45,98 @@ https://github.com/openshift/origin[origin repo].
 === Develop locally on your host
 
 You can develop OpenShift 3 on Windows, Mac, or Linux, but you'll need Docker
-installed on Linux to actually launch containers.
+installed on Linux to actually launch containers. Client and server binaries
+can be built locally or in the `openshift/release` container environemnt. The
+http://golang.org/[Go] programming language is only necessary for building on
+the local host.
```

Comment:
It should be noted that at least on Fedora, `golang-race` is needed or `make test-unit` fails with

```
TEST_KUBE=true GOTEST_FLAGS="-v" hack/test-go.sh  pkg/security/admission
[INFO] [CLEANUP] Cleaning up temporary directories
runtime/internal/sys
go install runtime/internal/sys: mkdir /usr/lib/golang/pkg/linux_amd64_race: permission denied
[ERROR] PID 25368: hack/test-go.sh:236: `go test -i ${gotest_flags} ${test_packages}` exited with status 1.
[INFO] 		Stack Trace: 
[INFO] 		  1: hack/test-go.sh:236: `go test -i ${gotest_flags} ${test_packages}`
[INFO]   Exiting with code 1.
```

Which specifically for Fedora 25 (the latest released) means that .fc26 builds from koji have to be installed manually.